ReadsUtils.spec

```diff
@@ -103,6 +103,15 @@ module ReadsUtils {
             single end reads.
         insert_size_std_dev - the standard deviation of the size of the
             genetic fragments. Ignored for single end reads.
+        sequencing_tech - the sequencing technology used to produce the
+            reads. (Is required if source_reads_ref is not specified)
```

Comment:
This should probably be higher up in the spec. I'd put it under the required section with a note like UNLESS source_reads_ref is specified.